data/infra/resources/apt_package.yaml

```diff
@@ -135,3 +143,11 @@ examples: |
     options '--no-install-recommends'
   end
   ```
+  **If a package has pattern like name, prevent installation of other apt packages which substring-matches the package's name**:
+
```

Comment:
```suggestion

  **Prevent the apt_package resource from installing packages with pattern matching names**:
  
  By default, the apt_package resource will install the named package.
  If it can't find a package with the exact same name, it will treat the package name as regex string and match with any package that matches that regex.
  This may lead Chef Infra Client to install one or more packages with names that match that regex.

  In this example, `anchor_package_name_patterns false` prevents the apt_package resource from installing matching packages if it can't find the `lua5.3` package.

```

Reply:
We have renamed the property as `anchor_package_regex` and by default it will now be false. Setting it to true will force single package name matching.